src/main/resources/assets/sodium/lang/en_us.json

```diff
@@ -4,6 +4,19 @@
     "speedrunapi.config.sodium.category.speedrun": "Speedrun",
     "speedrunapi.config.sodium.option.quality:enableVignette": "Vignette",
     "speedrunapi.config.sodium.option.quality:enableVignette.description": "If enabled, a vignette effect will be rendered on the player's view. This is very unlikely to make a difference to frame rates unless you are fill-rate limited.",
+    "speedrunapi.config.sodium.option.advanced:initialChunkThreads": "Initial Chunk Threads",
+    "speedrunapi.config.sodium.option.advanced:initialChunkThreads.description": "How many chunk building threads to create when reloading (lower = less F3 + F lag). Cannot be more than Target Chunk Threads.",
+    "speedrunapi.config.sodium.option.advanced:initialChunkThreads.value.0": "Auto",
+    "speedrunapi.config.sodium.option.advanced:targetChunkThreads": "Target Chunk Threads",
+    "speedrunapi.config.sodium.option.advanced:targetChunkThreads.description": "How many chunk building threads to build up to, initially (ideal: Number of hardware threads). Cannot be more than Maximum Chunk Threads.",
+    "speedrunapi.config.sodium.option.advanced:targetChunkThreads.value.0": "Auto",
+    "speedrunapi.config.sodium.option.advanced:maxChunkThreads": "Maximum Chunk Threads",
+    "speedrunapi.config.sodium.option.advanced:maxChunkThreads.description": "How many chunk building threads can ever be created (may help long-term chunk loading)",
```

Comment:
I think the wording of the option descriptions is a bit confusing, I`d suggest using the same wording where possible and only changing the key words, for example:
```
    "speedrunapi.config.sodium.option.advanced:initialChunkThreads.description": "How many chunk building threads to create instantly when reloading (lower = less F3 + F lag). Cannot be more than Target Chunk Threads.",
    "speedrunapi.config.sodium.option.advanced:targetChunkThreads.description": "How many chunk building threads to create quickly after reloading (ideal: Number of hardware threads). Cannot be more than Maximum Chunk Threads.",
    "speedrunapi.config.sodium.option.advanced:maxChunkThreads.description": "How many chunk building threads to create in total (may help long-term chunk loading)",
```